samples/net/google_iot_device/src/bsp/iotc_bsp_io_net_zephyr.c

```diff
@@ -198,9 +206,9 @@ iotc_bsp_io_net_state_t iotc_bsp_io_net_select(
   }
 
   /* call the actual posix select */
-  const int result = poll(fds, 1, timeout_sec);
+  const int result = poll(fds, socket_events_array_size, timeout_sec);
```

Comment:
Actually, the fds is still always 1 sized regardless of the incoming socket_events_array_size.

Reply:
> Actually, the fds is still always 1 sized regardless of the incoming socket_events_array_size.

`fds` is 1-sized, but if socket_events_array_size is 0, we should pass as much to poll().

Reply:
True, and 1< socket events size is handled above, ok, merging, squashing... :hourglass_flowing_sand:

Reply:
> ok, merging, squashing...

Thanks!

Reply:
Done, thanks for you efforts! +sorry for the delay.